portfolio.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+	<html lang="en-GB">
+	
+	<head>
+		<title>Charlotte Buckley - Portfolio</title>
```

Comment:
need to add in the line `<meta name="viewport" content="width=device-width, initial-scale=1.0">`